app/models/account/entry.rb

```diff
@@ -223,4 +225,8 @@ def create_trend
         previous: previous_entry&.amount_money,
         favorable_direction: account.favorable_direction
     end
+
+    def assign_transfer_name
+      self.name = transfer.name
+    end
```

Comment:
I think we should move this logic directly to the transfers controller:

https://github.com/maybe-finance/maybe/blob/490f44589e846e5b5d2b6bf56ddb4382234f0067/app/controllers/account/transfers_controller.rb#L18

Furthermore, it may be a nice touch if we had generated names for each of the entries within the transfer:

https://github.com/maybe-finance/maybe/blob/490f44589e846e5b5d2b6bf56ddb4382234f0067/app/models/account/transfer.rb#L46-L64

For example, the "from" name could be something like `"Transfer to #{to_account.name}"` and the "to" name could be `"Transfer from #{from_account.name}"`

Reply:
Yes , This makes sense. 
I believe if we’re assigning distinct names to both entries, then the `name` parameter in `build_from_accounts` function would be of no use, so removing it seems like the right approach to me

Also, Are we going to store the full name i.e Transfer from {from account} to {to account} elsewhere or this will going to be generated at runtime, like we do now.

Reply:
Yes , This makes sense.
I believe if we’re assigning distinct names to both entries, then the name parameter in build_from_accounts function would be of no use, so removing it seems like the right approach to me

Also, Are we going to store the full name i.e Transfer from {from account} to {to account} elsewhere or this will going to be generated at runtime, like we do now.

Reply:
Also, I've noticed that locales are already used in the project, so directly adding `"Transfer to #{to_account.name}"` might not be ideal. Would it be better to create a locale attribute under the `entry` folder in `en.yml`, or would that be overkill for this change?

Reply:
@Harry-kp regarding locales, we're going to have to go back through a lot of the model stuff and make bulk updates once we start down the path of #1225 , so for this change, let's just hardcode it for now.

And in regards to the `name:` param, fully agree.

> Are we going to store the full name i.e Transfer from {from account} to {to account} elsewhere

That's my bad, I forgot that `Transfer` does not have a stored `name` field.  We can just keep generating at runtime as we're doing now.

Reply:
@zachgoll  Changes are done.